### DISCUSSION TITLE
copying a project board to another organization yielding the error `There was a problem copying the project. Settings option description can't be nil`

### DISCUSSION THREAD

**Discussion by eleehiga** [community/community#46778] [Projects and Issues]
### Select Topic Area

Bug

### Body

On Github when trying to copy a Project board to another organization I get the error "There was a problem copying the project. Settings option description can't be nil". Is this a bug? Thank you.

**Comment by rasulkireev** [comment]
I don't have the same problem, but probably related.

https://github.com/community/community/discussions/46792

**Comment by rasulkireev** [comment]
The issue has been fixed in the discussion mentioned above.
